### DISCUSSION TITLE
2021/11/12/%E8%BF%99%E5%87%A0%E5%A4%A9%E5%9C%A8%E5%AD%A6%E4%B9%A0react

### DISCUSSION THREAD

**Discussion by giscus** [ruanqizhen/blog#360] [Announcements]
# 2021/11/12/%E8%BF%99%E5%87%A0%E5%A4%A9%E5%9C%A8%E5%AD%A6%E4%B9%A0react

之前几乎没做过任何网页开发的工作。不知道能不能用到建个人主页上。

https://blog.qizhen.xyz/2021/11/12/%E8%BF%99%E5%87%A0%E5%A4%A9%E5%9C%A8%E5%AD%A6%E4%B9%A0react

**Comment by ruanqizhen** [comment]
ruanqizhen
2021年11月16日 09:10 编辑
虽然还没全学完，但感觉上React更适合搭建有复杂交互操作的网站，比如要做个在线图片编辑网站。
个人博客也用不着什么复杂交互，主要是展示内容的。一是没必要用React，另外听说一些中文搜索引擎对React页面支持不好，检索不了React网页提供的内容。所以中文博客用React不是个好选择。

回复
